pipeline/console/__init__.py

```diff
@@ -231,6 +233,140 @@ def main(args: Optional[List[str]] = None) -> int:
     )
 
     ##########
+    # pipeline environments
+    ##########
+
+    environments_parser = command_parser.add_parser(
+        "environments",
+        description="Manage environments",
+        help="Manage environments",
+    )
+
+    environments_sub_parser = environments_parser.add_subparsers(dest="sub-command")
+
+    ##########
+    # pipeline environments create
+    ##########
+
+    environments_create_parser = environments_sub_parser.add_parser(
+        "create",
+        help="Create a new python environment",
+    )
+
+    environments_create_parser.add_argument(
+        "name",
+        help="The environment name",
+    )
+
+    # TODO: Create via requirements.txt
```

Comment:
Looks like this might be supported by `_get_packages_from_requirements`?

Reply:
Yeah forgot to rm this, ty